vars/perfRegressionParallelPipeline.groovy

```diff
@@ -74,129 +111,44 @@ def call(Map pipelineParams) {
                             } else {
                                 perf_test = "${pipelineParams.test_name}.${sub_test}"
                             }
-
+                            test_params["${sub_test}"] = params + ["test_name":"${perf_test}", "test_id":UUID.randomUUID().toString()]
```

Comment:
instead all this keeping of the varius test_params in mapping,
why not using `withEnv` to pass down the correct test id ? to each split ?